### ISSUE TITLE
unitPixels variable of the RAM model

### ISSUE THREAD

**Issue by vyouman** [Element-Research/rnn#167]
Hi, thanks for your nice work.
I want to apply the RAM model to some real-word dataset, say there is wide variation in image size across the dataset. 
And there's a important variable which you mentioned in your previous blog post. I think the variable is designed to constrain the center of the image patches in order to reduce the zero paddings of the glimpses located in the corner, so when applied to images of variable size, it can probably be ommitted I guess.

> -- actions (locator)
> locator = nn.Sequential()
> locator:add(nn.Linear(opt.hiddenSize, 2))
> locator:add(nn.HardTanh()) -- bounds mean between -1 and 1
> locator:add(nn.ReinforceNormal(2 * opt.locatorStd)) -- sample from normal, uses REINFORCE learning rule
> locator:add(nn.HardTanh()) -- bounds sample between -1 and 1
> locator:add(nn.MulConstant(opt.unitPixels * 2/ds:imageSize("h")))

So should I resize all these images to the same size and then specify some value for the unitPixels variable, or just simply ignore the variable? Can you give me some suggestions? Many thanks.

**Comment by nicholas-leonard** [comment]
@vyouman You should resize all your images to the same size (so you can batch them together) and use a `unitPixels` that minimizes overlap with padding. Good value depends on glimpseSize and imageSize. You can use the script's default as a reference point. Also use the --stochastic switch. Forgot to mention that in the blog post.
